src/database/sqlite/sqlite_metadata.c

```diff
@@ -67,6 +67,18 @@ const char *database_config[] = {
     "CREATE INDEX IF NOT EXISTS health_log_d_ind_7 on health_log_detail (alarm_id)",
     "CREATE INDEX IF NOT EXISTS health_log_d_ind_8 on health_log_detail (new_status, updated_by_id)",
 
+#ifdef ENABLE_ACLK
```

Comment:
These tables do not have `default` value for fields, I suggest we add them avoiding issues if a developer forgets to set one of them.

Reply:
health_log_detail is not touched (altered) by this PR. 

Similarly you can notice that the entire `alert_version` table could be omitted by just adding some fields to `health_log`.  Maybe in a future PR given the fact that ACLK will always be available.